File: problems/3238-find-the-number-of-winning-players/solution.c
int winningPlayerCount(int n, int** pick, int pickSize, int* pickColSize) {
    int wins = 0;
    for (int player = 0; player < n; player++) {
        int counts[11] = {0};
        for (int i = 0; i < pickSize; i++) {
            if (pick[i][0] == player) {
                counts[pick[i][1]]++;
            }
        }
        for (int i = 0; i < 11; i++) {
            if (counts[i] > player) {
                wins++;
                break;
            }
        }
    }
    return wins;
}

File: problems/3238-find-the-number-of-winning-players/solution.py
class Solution:
    def winningPlayerCount(self, n: int, pick: List[List[int]]) -> int:
        wins = 0
        for i in range(n):
            counts = {}
            for p in pick:
                if p[0] == i:
                    if p[1] not in counts:
                        counts[p[1]] = 0
                    counts[p[1]] += 1
            won = False
            for color in counts:
                if counts[color] > i:
                    won = True
                    break
            if won:
                wins += 1
        return wins